task/src/task.test.ts

```diff
@@ -80,4 +92,17 @@ describe('Integration Test', () => {
         expect(tl.setResult).toMatchSnapshot();
         expect(gitApiMock.createThread).toMatchSnapshot();
     });
+
+    it('should use provided newCommentSeparator to split comments', async () => {
+        const newCommentSeparator = "||";
+
+        (tl.getInput as any)
+            .mockReturnValueOnce(`comment one${newCommentSeparator}comment two${newCommentSeparator}comment three`)
```

Comment:
Maybe throw in a comment with a few \ns to make sure that's it's only using newCommentSeparator to split comments?

Reply:
Good idea, will do

Reply:
Added, lmk if it suffices